src/Streamly/Internal/Data/Fold.hs

```diff
@@ -19,10 +19,10 @@ module Streamly.Internal.Data.Fold
     , Fold (..)
 
     -- * Constructors
-    , mkFoldl
-    , mkFoldlM
+    , foldl'
+    , foldlM'
     , mkFoldl1
-    , mkFoldr
+    , foldr
     , mkFoldrM
```

Comment:
We can change `mkFoldl1` to `foldl1` and `mkFoldrM` to `foldrM` at the same time. Only `mkFold` and `mkFold_` should remain unchanged.